### ISSUE TITLE
typo README.md 

### ISSUE THREAD

**Issue by secdev-01** [center-for-threat-informed-defense/attack-flow#36]
```
In the attack_flow_builder directory
```

s/b 

```
In the attack_flow_designer directory
```

![Screen Shot 2022-03-19 at 7 46 45 AM](https://user-images.githubusercontent.com/83469949/159123633-b4d51695-977e-4fa7-a5e3-89af3f4d85a7.png)

**Comment by mehaase** [comment]
Thank you for pointing this out. I'll update it asap.

**Comment by mehaase** [comment]
Closed via #37
